src/cr-logging/Logging.cs

```diff
@@ -0,0 +1,110 @@
+﻿// <copyright file="Logging.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging
+{
+    using System;
+    using System.Configuration;
+    using Microsoft.Extensions.Configuration;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using Serilog.Formatting;
+
+    /// <summary>
+    /// The Logging helper class which can be used to initialize a pre-configured Serilog Logger instance.
+    /// </summary>
+    // ReSharper disable once UnusedMember.Global
+    public static class Logging
+    {
+        private static IConfigurationRoot _appConfig;
```

Comment:
This variable is only used during the execution of `SetupLogger` so would be better passed into any method that uses it.